### DISCUSSION TITLE
Dropzone input props to support directory property on <input>

### DISCUSSION THREAD

**Discussion by volkandkaya** [mantinedev/mantine#4123] [Feature requests]
https://stackoverflow.com/questions/12942436/how-to-get-folder-directory-from-html-input-type-file-or-any-other-way

You can drop a folder on it, but can't upload a folder if you click the dropzone.

**Comment by dylangrandmont** [comment]
This would be very useful to have in the `mantine/dropzone` package. Some prior art exists for handling drag-and-drop for directories. For example, https://github.com/dropzone/dropzone
